docs/data/toolpad/core/components/account/account.md

```diff
@@ -26,12 +26,16 @@ When signed out, the component renders as an inline sign in button within the da
 
 ## Customization
 
-### Components
+### Slot Props
 
-`Account` can take different labels for the sign in and sign out buttons through the `signInLabel` and `signOutLabel` props. Deeper changes can be made by passing in `slotProps` to the underlying components.
+The underlying `signInButton`, `signOutButton` and `iconButton` components can be customized by passing in `slotProps` to the `Account` component.
+
+Labels for the sign in and sign out buttons through the `localeText` prop.
```

Comment:
I think this sentence about the labels is missing something ("can be customized?").